requirements/requirements_nlp.txt

```diff
@@ -22,4 +23,5 @@ sacrebleu  # manually install sacrebleu[ja] for Japanese support; MeCab is unsup
 sentence_transformers
 tensorstore
 tiktoken==0.7.0
+unstructured==0.14.9
```

Comment:
Same with this - https://github.com/Unstructured-IO/unstructured 
This is actually even heavier than the above somehow.. https://github.com/Unstructured-IO/unstructured/tree/main/requirements 

@ericharper